src/NUnitTestAdapter/NUnitTestAdapter.cs

```diff
@@ -131,16 +134,43 @@ protected void Initialize(IDiscoveryContext context, IMessageLogger messageLogge
             {
                 Settings.Load(context);
                 TestLog.Verbosity = Settings.Verbosity;
-               
+                InitializeForbiddenFolders();
+                SetCurrentWorkingDirectory();
             }
             catch (Exception e)
             {
                 TestLog.Warning("Error initializing RunSettings. Default settings will be used");
                 TestLog.Warning(e.ToString());
             }
+            
+        }
+
+        public void InitializeForbiddenFolders()
+        {
+            ForbiddenFolders = new []
+            {
+                Environment.GetEnvironmentVariable("ProgramW6432"),
+                Environment.GetEnvironmentVariable("ProgramFiles(x86)"),
+                Environment.GetEnvironmentVariable("windir"),
+            }.Where(o => !string.IsNullOrEmpty(o)).Select(o=>o.ToLower()+@"\").ToList();
```

Comment:
```suggestion
            }.Where(o => !string.IsNullOrEmpty(o)).Select(o=>o + @"\").ToList();
```

Reply:
Other than this, LGTM!

Reply:
@OsirisTerje Was there a reason you needed ToLower()?

Reply:
The ToLower could be removed here too, you're right.  Next time around, as I did the merge :-)